lib/src/doc/lives.rs

```diff
@@ -100,42 +96,160 @@ impl<'a> Document<'a> {
 				// Finally, let's check what type of statement
 				// caused this LIVE query to run, and send the
 				// relevant notification based on the statement.
+				let mut tx = txn.lock().await;
+				let ts = tx.clock().await;
+				let ns = Box::new(opt.ns().to_string());
+				let db = Box::new(opt.db().to_string());
+				let tb = Box::new(self.id.unwrap().tb.to_string());
+				let not_id = crate::sql::Uuid::new_v4();
 				if stm.is_delete() {
 					// Send a DELETE notification
+					let thing = (*rid).clone();
+					let notification = Notification {
+						live_id: lv.id.clone(),
+						node_id: lv.node.clone(),
+						notification_id: not_id.clone(),
+						action: Action::Delete,
+						result: Value::Thing(thing),
+						timestamp: ts.clone(),
```

Comment:
If we make `Timestamp` `Copy` we can remove this clone too.

Reply:
Will do thanks